codewars/rackunit.rkt

```diff
@@ -16,35 +16,63 @@
 
 ;;; handle failed test output
 (define (log-exn:test:check e mode)
-  (with-tag "FAILED"
-    (check-infos->string
-     (check-info-stack-filter (exn:test:check-stack e) mode))))
-
-
+  (define stack (exn:test:check-stack e))
+  (define content
+    (cond
+      [(eq? mode 'quiet) (check-infos->string/quiet stack)]
+      [else (check-infos->string
+             (check-info-stack-filter stack mode))]))
+  (with-tag "FAILED" content))
+
+;;; 'quiet mode
+(define (check-infos->string/quiet stack)
+  (define (get-info-value/string name)
+    (info-value->string (get-info-value stack name)))
+  (define (get-info-value/string* name)
+    (let ([value (get-info-value stack name)])
+      (cond
+        [(eq? value #f) #f]
+        [(string? value) (format "~a" value)]
+        [else (info-value->string value)])))
+  
+  (define expected (get-info-value/string 'expected))
+  (define actual (get-info-value/string 'actual))
+  (define message (get-info-value/string* 'message))
+
+  (apply string-append
+         "Expected " expected
+         ", but instead got " actual
+         (if message
+             (list "\nMessage: " message)
```

Comment:
Remove `"\nMessage: "`? I think it's obvious it's some extra message and makes a message with multiple lines look awkward.

![image](https://user-images.githubusercontent.com/639336/54942263-f1a64e80-4eeb-11e9-93d2-12c0953f3ca9.png)